template/docroot/sites/default/settings/files.settings.php

```diff
@@ -0,0 +1,12 @@
+<?php
+
+/**
+ * @file
+ * Contains Files configuration.
+ */
+if ($is_acsf) {
+  $settings['file_private_path'] = "/mnt/gfs/$ah_group.$ah_env/files-private/$acsf_db_name";
+}
+else {
+  $settings['file_private_path'] = "/mnt/files/$ah_group.$ah_env/files-private";
```

Comment:
We can use `$site_path` instead of `$conf_path` (the ACE settings files provide this variable.)

The bigger problem is how does this file know if we area single-site or multisite install? We probably need a parameter in `project.yml`, as well as a way to propagate that variable into settings files.